docs/UserGuide.md

```diff
@@ -142,11 +159,106 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
-### Clearing all entries : `clear`
+### Assigning dates to a contact: `contacted`
+
+Manually logs the date of the most recent interaction with the contact.
+
+Format: `contacted INDEX d/DATE n/description`
+* Dates are in `DD-MM-YYYY` format.
+
+Example:
+* `contacted 23 d/28-07-2022 n/Sign Contract`
+
+### Displaying contacted contacts within DAYS: `within`
+
+Displays contacted contacts between **now**, and a **specified date**
+
+Format: `within d/DAY`
+* Dates are in `DD-MM-YYYY` format.
+
+Example:
+* `within d/11-09-2022`
+
+### Displaying contacted contacts later than DAYS: `later`
+
+Displays contacted contacts **after a specified date**.
+
+Format: `later d/DAY`
+* Dates are in `DD-MM-YYYY` format.
+
+Example:
+* `later d/29-02-2024`
+
+### Adding a non-recurring reminder to a contact: `remind`
+
+Creates a reminder for a contact.
+
+Format: `remind INDEX r/REMINDER d/DATE`
+* Dates are in `DD-MM-YYYY` format.
+
+Example:
+* `remind 13 r/dinner d/12-03-2022`
+
+
+### Viewing reminders of a contact: `reminder`
+
+Displays all reminders under a contact as specified by the index.
 
-Clears all entries from the address book.
+Format: `reminder INDEX`
 
-Format: `clear`
+Example:
+* `reminder 7`
+
+### Viewing contacts by reminder: `reminders`
+
+Displays all contact with a reminder with the specified date.
+
+Format: `reminders [d/DATE]`
+
+Example:
+* `reminders d/01-01-2023`
+
+### Deleting a reminder: `forget`
+
+Deletes the specified reminder of a contact as specified by the index.
+
+Format: `forget INDEX r/REMINDER`
+
+Example:
+* `forget 5 r/Dinner outing`
+
+### Adding a tag: `tag`
+
+Adds a tag to an existing contact, as specified by the index.
+
+Format: `tag INDEX #tag`
+
+Example:
+* `tag 27 #Cereal Killer`
```

Comment:
I think we should `t/Cereal`, and the current version of tag does not allow spaces, only alphanumeric. Same for others.